core/themes/seven/css/style.css

```diff
@@ -799,7 +799,7 @@ details {
   background-color: #ffffff;
   display: table-cell;
 }
-details summary span {
+details:not(.more-less-toggle) summary span {
```

Comment:
I anticipate people asking why this change here. If we don't exclude the "special" more/less details elements from this, then we have to specifically override this font color/size each time we use `<details>` in:
- help text (which needs to be light gray)
- the modules listing (which needs to be black)
- the status page (needs to be black)
- ...any other place where blue text for the details summary is not required

Reply:
I've added an inline comment to explain that. Resolving this comment here.